src/detections/rule/selectionnodes.rs

```diff
@@ -611,4 +617,46 @@ mod tests {
 
         check_select(rule_str, record_json_str, false);
     }
+
+    #[test]
+    fn test_detect_event_id_wildcard() {
+        // EventIDのワイルドカードマッチが正しく検知することを確認する。
```

Comment:
EventID detection should not accept regular expressions (wildcards), but isn't it a requirement for detection in this test?

Reply:
@hitenkoku Thanks for letting us know about this test. I checked all the sigma rules and no rules use wildcards in the EventID and I can't think of a situation where it would be needed so it is probably better to remove this test and do exact matches. What do you think?

Reply:
@hitenkoku Ah, Fukusket's PR does fallback to using regular expressions if needed, so we can probably just keep the test and support wildcards even though it is probably not necessary. what do you think? If it doesn't affect performance, we might as well support regular expressions when needed but do exact string matches by default.

Reply:
ok I resolve this conversation.